languages/csharp/exercises/concept/lists/ListsTests.cs

```diff
@@ -11,7 +11,11 @@ public void NewList()
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void ExistingList()
     {
-        Assert.Equal(new string[] {"C#", "Clojure", "Elm"}, Languages.GetExistingLanguages());
+        var expected = new List<string>();
```

Comment:
BTW This also applies to the other tests that use the array (I was too lazy too add suggestions for the others :))

Reply:
sure. np